regressiontests/vet_shadow_test.go

```diff
@@ -1,3 +1,5 @@
+// +build !go1.9,go1.10
```

Comment:
Instead of using build tags to select tests could we use `runtime.Version()` to skip the test on 1.8?

```go
if runtime.Version() == "go1.8" {
    t.Skip("go vet does not have a --shadow flag in go1.8")
}
```